bundles/org.eclipse.osgi.tests/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,8 @@ Require-Bundle:
  org.eclipse.core.runtime,
  org.eclipse.core.tests.harness,
  org.eclipse.test.performance,
- org.junit
+ org.junit,
+ org.eclipse.osgi.util;bundle-version="3.7.0";resolution:=optional
```

Comment:
Is there an explanation on why this was needed vs using `Import-Package`?

Reply:
I just did that to pull in the requirements into Tycho-surefire's test-runtime.
But I assume this is better done by specifying extraRequirements in the pom.xml.